nix/wrapper.nix

```
@@ -3,12 +3,14 @@
 let
   mkCommand = name: v:
     let
-      drv = pkgs.writeShellApplication { inherit name; text = if builtins.typeOf v.exec == "string" then v.exec else ''${lib.getExe v.exec} "$@"''; };
+      drv = pkgs.writeShellApplication { inherit name; text = if builtins.typeOf v.exec == "string" then v.exec else ''${lib.getExe v.exec} "$@"'';};
     in
     drv.overrideAttrs (oa: {
       meta.description =
         if v.description == null then oa.meta.description or "No description" else v.description;
       meta.category = v.category;
+      meta.mainProgram =
+        if oa.meta.mainProgram == null then v.name or "No name" else oa.meta.mainProgram;
```

Any reason why this can't simply be:

```suggestion
      meta.mainProgram =
        oa.meta.mainProgram or v.name;
```

?